Packs/CyberArkIdentity/Integrations/CyberArkIdentityEventCollector/CyberArkIdentityEventCollector_description.md

```diff
@@ -5,16 +5,22 @@ Before you start, you need to get the tenant ID, app ID, username, and password.
 ## Configuration Parameters
```

Comment:
The first line should be (Removed the "Event Collector"):
Use the CyberArk Identity integration to get Audit and Auth logs for an application using REST APIs.